benchmark/benchmark.awk

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2023 Fotios Valasiadis
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+BEGIN {
+    FS = ":"
+    print "Baseline run | Build-recorder run | Comparison %"
+    print
+}
+
+NR == FNR {
+    # Build recorder run
+    brrun[$1] = $2
+    next
+}
+
+{
+    if ($1 == "CPU utilization") {
```

Comment:
Well you asked to just change the formula above so I'll just do that for now.

You are correct that it is easy to compute it, only reason to keep it is convenience, not having to!

Reply:
After all it depicts the most important metric I used to optimize build-recorder a while ago, CPU utilization, once we run a build with multiple threads it will be revealed.